client/app/nasha/partition/access/nasha-partition-access.html

```diff
@@ -1,39 +1,35 @@
-        </a>
 
-        <h3 class="bottom-space-m12" data-translate="nasha_partitions_action_change_access"></h3>
+    <oui-back-button data-title="{{::'nasha_partitions_action_change_access' | translate}}"
+        data-on-click="NashaPartitionAccessCtrl.$state.go('paas.nasha.nasha-partitions')"></oui-back-button>
 
-        <h4 class="bottom-space-m8" data-translate="nasha_access_partition" data-translate-values="{partitionName: NashaPartitionAccessCtrl.data.partition.partitionName}"></h4>
+    <p class="bottom-space-m8" data-translate="nasha_access_partition" data-translate-values="{partitionName: NashaPartitionAccessCtrl.data.partition.partitionName}"></p>
 
-        <button type="button"
-            class="oui-button oui-button_secondary"
-            data-ng-disabled="NashaPartitionAccessCtrl.loaders.table"
-            data-ng-click="NashaPartitionAccessCtrl.addAccess()"
-            data-translate="nasha_access_action_add"></button>
+    <button type="button"
+        class="oui-button oui-button_secondary"
+        data-ng-disabled="NashaPartitionAccessCtrl.loaders.table"
+        data-ng-click="NashaPartitionAccessCtrl.addAccess()"
+        data-translate="nasha_access_action_add"></button>
 
-        <oui-datagrid data-rows="NashaPartitionAccessCtrl.table.accessIps"
-            data-row-loader="NashaPartitionAccessCtrl.transformItem($row)">
-            <oui-column data-property="ip"
-                data-title="'nasha_access_ip' | translate"
-                data-sortable="asc"></oui-column>
-            <oui-column data-property="type"
-                data-title="'nasha_partitions_access_permissions' | translate"
-                data-sortable>
-                <span data-ng-if="$row.type">{{ 'nasha_partitions_access_' + $row.type | translate }}</span>
-            </oui-column>
-            <oui-column>
-                <oui-spinner data-size="s" data-ng-show="NashaPartitionAccessCtrl.hasTaskInProgress($row)"></oui-spinner>
-            </oui-column>
-            <oui-action-menu data-align="end" data-compact>
-                <oui-action-menu-item data-text="{{ 'common_remove' | translate }}"
-                    data-on-click="NashaPartitionAccessCtrl.removeAccess($row)"
-                    data-disabled="NashaPartitionAccessCtrl.hasTaskInProgress($row)">
-                </oui-action-menu-item>
-            </oui-action-menu>
-        </oui-datagrid>
+    <oui-datagrid data-rows="NashaPartitionAccessCtrl.table.accessIps"
+        data-row-loader="NashaPartitionAccessCtrl.transformItem($row)">
+        <oui-column data-property="ip"
+            data-title="'nasha_access_ip' | translate"
+            data-sortable="asc"></oui-column>
+        <oui-column data-property="type"
+            data-title="'nasha_partitions_access_permissions' | translate"
+            data-sortable>
+            <span data-ng-if="$row.type">{{ 'nasha_partitions_access_' + $row.type | translate }}</span>
```

Comment:
It's not strange to have a first element, with only one tag as child, and a ng-if on it? If the ngIf is false, we have an empty cell?

Maybe it's logical, but i prefer to ask ^^

Reply:
Thanks.